crates/abi-conformance/src/outbound_redis.rs

```diff
@@ -76,6 +77,15 @@ pub struct RedisReport {
     /// "baz"))` as the result.  The host will assert that said function is called exactly once with the specified
     /// arguments.
     pub smembers: Result<(), String>,
+
```

Comment:
As a total aside, it would be interesting to run these conformance tests under tarpaulin.